src/manage_sql.c

```diff
@@ -54745,12 +54745,11 @@ modify_user (const gchar * user_id, gchar **name, const gchar *new_name,
   if (allowed_methods && (allowed_methods->len > 2))
     return -3;
 
-  if (allowed_methods && (allowed_methods->len == 0))
+  if (allowed_methods && (allowed_methods->len <= 0))
```

Comment:
Yes, thank you, must have messed up my testing because it definitely segfault with 0. Added in [1cf3629](https://github.com/greenbone/gvmd/pull/2207/commits/1cf3629fcdbc405a554f048701f1a9634641e9c5).